Vester/Tests/Network/Vswitch-config.Vester.ps1

```diff
@@ -0,0 +1,37 @@
+# Test file for the Vester module - https://github.com/WahlNetwork/Vester
+# Called via Invoke-Pester VesterTemplate.Tests.ps1
+
+# Test title, e.g. 'DNS Servers'
+$Title = 'vswitch0 details'
+
+# Test description: How New-VesterConfig explains this value to the user
+$Description = 'add vlan to vswitch0'

```

Comment:
This script and the vswitch mtu should maybe be in the same file. Additionally, this is hard coding vswitch0, but what if the user wants more than one vswitch? vswitches should probably be parameterized in the configuration and all of them configured through this script. I hope this makes sense.